easybuild/easyconfigs/i/icc/icc-2016.1.150-GCC-4.9.3.eb

```diff
@@ -8,7 +8,7 @@ description = "C and C++ compiler from Intel"
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
-sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_cpp_%(version)s.tgz']
+sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_cpp_update%(version_minor)s.tgz']
```

Comment:
What about original parallel_studio_xe_%(version_major)s_composer_edition_for_cpp.tgz? How about this:

```
sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_cpp_update%(version_minor)s.tgz',
                  'parallel_studio_xe_%(version_major)s_composer_edition_for_cpp.tgz'
]
```
Based on MD5 will be selected right one?